CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 #### Enhancements
 
+* Added `method` parameter to `HTTP.Request`.ß
```

Comment:
Is the extra character at the end needed?

Reply:
Nope, went ahead and removed it